modules/network/nat-gateway/main.bicep

```diff
@@ -0,0 +1,60 @@
+@description('Required. The name of the NAT Gateway resource.')
+param name string
+
+@description('Required. Location(region) for NAT Gateway will be deployed.')
+param location string
```

Comment:
```suggestion
@description('Required. Location(region) for NAT Gateway will be deployed.')
param location string

@description('Prefix of NAT Gateway Resource Name. This param is ignored when name is provided.')
param prefix string = 'ng'

@description('Optional. The name of the NAT Gateway resource.')
param name string = '${prefix}${uniqueString(resourceGroup().id, subscription().id)}'

```

Reply:
i like to give name a default value, using a prefix from https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations